### DISCUSSION TITLE
[pvp] Soloq needs some tweaking

### DISCUSSION THREAD

**Discussion by tbcxd** [chromiecraft/chromiecraft#7494] [Suggestions]
So far the soloq has been fun, but I've ran into issues where we get matched up as 3 dps vs a team with 2 healers.

I'm not a coding expert, but soloq needs to have some conditions implemented

Such as:
**No game pop if there is no healer queued** 3 dps games are not fun.
**No more than 1 healer per team** same issue as the 3 dps, healers can be matched in the same team vs a team with no healer.

A daily 20 arena points for 10x soloq games would also be sweet as  season3 will be shorter, but this might take too much time to implement and can be done at a later time

**Comment by TotallyNotMyAccount** [comment]
I had a 3 heal vs 3 DPS game yesterday. This stuff needs work for sure.

**Comment by Belodina** [comment]
There was some good work put into battleground queue rules, and now joining a battleground for example almost always leads to balanced teams number wise.

However, it has been the case that through RNG, heals are queued into one side, leading to a WSG with 6/10 healers on one team, vs a team full of DPS. 

Since the 3v3 arena queue can differentiate between heal/DPS specs when viewing at the 3v3soloq NPC, and if it is possible to enforce role limitations for better balanced matchmaking, would it be possible to do the same for battleground queue? i.e. if there is a healer on one team, and not on the other team, it will prioritize putting the healer into the team without one and so on and so forth.

Otherwise, been having a blast in soloq even with the wonky matches, but for sustainability and long-term enjoyment some matchmaking rules would go a long way.

**Reply by Helias** [reply]
It is possible to balance the players in BGs and/or in 3v3soloQ.
In the past, I've implemented a balance  for low-level BGs considering the Hunter class a special one to  balance in both side and also consider the difference of the level to avoid having two lvl19 against two lvl10.

I hope we can work on a better balance for 3v3soloQ and BGs.
